requirements.txt

```diff
@@ -54,3 +54,4 @@ deepdiff~=5.0
 pymysql~=1.0
 inflection~=0.5.0
 python-dotenv~=0.17.0
+pymongo~=4.1.1
```

Comment:
I don't understand why it should be added to everyone, in my eyes its like I would put xgboost, lightgbm, catboost, tensorflow and pytorch in the requirements.

Reply:
also, while we are at it, you did used `bason` or something like that as well no? so why no to add this as well?

Reply:
So where you think I should write this requirements? @guy1992l